setup.py

```diff
@@ -21,13 +21,13 @@
   long_description=long_description,
   long_description_content_type='text/markdown',
   install_requires=[
-    'requests'
+    'requests>=2.31.0'
   ],
   setup_requires=[
-    'pytest_runner'
+    'pytest_runner==6.0.0'
```

Comment:
I see requirements.txt match this, but wouldn't `~=` be more flexible to account for patch version changes